packages/core/src/utils/serializeNode.tsx

```diff
@@ -17,7 +17,8 @@ export const serializeComp = (
   let { type, isCanvas, props } = data;
   props = Object.keys(props).reduce((result: Record<string, any>, key) => {
     const prop = props[key];
-    if (!prop) {
+
+    if (prop == null) {
```

Comment:
Good catch! 💯 Can you use strict equality `===` instead?

Reply:
Sure. Is triple equals the right way here? == will check against undefined, not just null.

Reply:
I mean something a bit more verbose like `prop === undefined || prop === null`. I think this makes the code a little bit more readable and potentially less bug prone.